src/views/common/DashOverview.tsx

```diff
@@ -31,13 +61,24 @@ export const DashOverviewIntro: React.FC<{
         currency={asset.symbol}
         amount={amount}
         setAmount={setAmount}
+        healthFactor={newHealthFactor}
         mode={mode}
-        balance={balance}
+        balance={
+          // If NATIVE don't allow to deposit the full balance in a wallet! Leave 0.1
+          asset.tokenAddress === NATIVE_TOKEN &&
+          (mode === "deposit" || mode === "repay")
+            ? limitAmount?.sub(BigNumber.from(10).pow(17))
```

Comment:
Define a module-level `export const MINIMUM_NATIVE_RESERVE = constants.WeiPerEther.div(10);` and use that here.